src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -580,8 +580,8 @@ public async Task<IActionResult> EnterNewUkprn(EnterNewUkprnViewModel model)
                     UKPRN = ukprn,
                     UkrlpLookupDetails = ukrlpLookupResults.Results.FirstOrDefault()
                 };
-
-                await _applicationApiClient.UpdateApplicationStatus(model.ApplicationId, ApplicationStatus.Cancelled);
+                var userName  = User.Identity.Name;
```

Comment:
It's wrong to assume this. We don't use names but Guids.

I think it should be User.GetUserId()